### PR TITLE
Feature blog article meta tags

### PR DESCRIPTION
**Pull request by ebeutler**
https://synjira.atlassian.net/browse/CELDEV-389

First merge https://github.com/celements/celements-core/pull/104

### CODE REVIEW

File: src/main/webapp/templates/celMacros/includeDefaultHeader.vm [hunk 1]
```diff
@@ -95,3 +94,6 @@ $!celementsweb.getAllExternalJavaScriptFiles()
 ## - Skin extension hook
 <!-- com.xpn.xwiki.plugin.skinx.JsSkinExtensionPlugin -->
 ##
+#if('Article' == "$!services.pageType.getPageTypeRef($doc.getDocumentReference()).getConfigName()")
```
Comment (fpichler) [inline src/main/webapp/templates/celMacros/includeDefaultHeader.vm]: please keep displayCollectedMetaTags and add a beforeDisplayCollectedMetaTag event which might be observed by any listener and add additional mediaTag elements before they get displayed. In general please do not add any strong cupling between any extension and celements-web

**Review by fpichler — CHANGES_REQUESTED** [review]

File: src/main/webapp/templates/celMacros/includeDefaultHeader.vm (hunk 1 above)
Comment (msladek) [reply src/main/webapp/templates/celMacros/includeDefaultHeader.vm]: agree, please implement according to agreed upon design

**Review by msladek — CHANGES_REQUESTED** [review]

**Comment by ebeutler** [comment]
Refactored.
Core changes on https://github.com/celements/celements-core/pull/104

**Review by msladek — APPROVED** [review]

**Comment by msladek** [comment]
looks neat now, thanks!

**Review by fpichler — APPROVED** [review]

**Pull request merged**